apps/frontend/src/components/global/checklist/ChecklistSeverityOverride.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-card class="text-center px-8 pt-2" height="300px">
+    <v-card-title class="justify-center"
+      >Severity Override Justification</v-card-title
+    >
+    <v-card-subtitle
+      v-if="selectedRule.severityJustification === ''"
+      class="justify-center mt-1"
+    >
+      <strong>
+        Please input a valid severity override justification. (Required)
+      </strong>
+    </v-card-subtitle>
+    <v-card-subtitle v-else class="justify-center mt-1">
```

Comment:
i think this clause can be removed.  it should be fine to have 'please input a blah' the entire time.